homebrew/install.sh

```diff
@@ -1,10 +1,14 @@
 #!/usr/bin/env bash
 
+LINK_DIR=$1
```

Comment:
Should this argument be optional? I don't understand where it comes from.

Reply:
It's been a while since I looked at this. I believe the install script is only used by homebrew, so it should be OK to require the argument. The idea behind this is that when the user installs nvs version 1.6.1, for example, homebrew saves the install in a `nvs/1.6.1/nvs-1.6.1` folder, and then if the user upgrade from version 1.6.1 to version 1.6.2, homebrew will install the new version to `nvs/1.6.2/nvs-1.6.2`, which means the `NVS_ROOT` variable that was previously set up to `nvs/1.6.1/nvs-1.6.1` won't work. The solution is to link the latest version to a `nvs-latest` folder and have the `NVS_ROOT` variable point to `nvs-latest`. This is why I'm adding the LINK_DIR.